extensions/adobe/experience/analytics/experienceevent-all.schema.json

```diff
@@ -5,12 +5,14 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/experience/analytics/experienceevent",
+  "$id": "https://ns.adobe.com/experience/analytics/experienceevent-all",
```

Comment:
since this is the extension that wraps everything intended to be added to the final schema, should we call it "extension" versus "all"?

Reply:
No, the experience/analytics-experienceevent.schema.json is the is the thing "that wraps everything intended to be added to the final schema". it is in-fact a base schema that Analytics will allOf to create the tenant level schema. e.g https://github.com/adobe/xdm/pull/554/files#diff-f452c02ba041146a1fb27a2f8be049bd 

The "-all" in a experience/{solution} is all the solution specific fields that are never shared.